src/list.scss

```diff
@@ -101,6 +101,107 @@ $block: #{$fd-namespace}-list;
         background: var(--sapList_Background);
       }
     }
+
+    &--action {
+      display: -webkit-box;
+      display: flex;
+      size: 2.5rem;
+      padding: 0 0.25rem;
+      -webkit-box-pack: center;
+      justify-content: center;
+      cursor: pointer;
+      border: var(--sapPageFooter_BorderColor);
+      background: var(--sapPageFooter_Background);
+      height: 3rem;
+
+      @include fd-ellipsis();
+
+      &--compact {
+        size: 1.625rem;
+        font-size: var(--sapFontSize);
+      }
+
+      .#{$block}__title {
+        -webkit-box-flex: 1;
+        flex: auto;
+        width: auto;
+        min-width: 3rem;
+        display: block;
+        text-align: center;
+        font-weight: normal;
+        color: var(--sapButton_TextColor);
+        font-size: var(--sapFontLargeSize);
+        text-shadow: none;
+
+        @include fd-ellipsis();
+        @include fd-rtl();
+      }
+    }
+
+    &--display {
+      display: -webkit-box;
+      display: flex;
+      -webkit-box-flex: 1;
+      flex: 1 1 auto;
+      max-width: 100%;
+
+      @include fd-ellipsis();
+
+      &--action {
+        cursor: pointer;
+      }
+
+      .#{$block}__title {
+        -webkit-box-flex: 1;
+        flex: auto;
+        min-width: 3rem;
+        display: block;
+        font-size: var(--sapFontLargeSize);
+        -webkit-box-align: center;
+        align-items: center;
+        color: var(--sapList_TextColor);
+        font-weight: normal;
+
+        @include fd-ellipsis();
+
+        @include fd-rtl() {
+          text-align: right;
+        }
+      }
```

Comment:
Please refer to the other comment regarding title. You inherit the code from the Standard list and only modify values that are specific for the case

Reply:
1).Placed mixins on top.
2). Removed -WebKit related code.
3).font-size and font-weight removed.